src/main/java/uk/gov/hmcts/reform/sendletter/entity/LetterRepository.java

```diff
@@ -14,7 +16,12 @@
 
     Stream<Letter> findByStatus(LetterStatus status);
 
-    Stream<Letter> findByStatusAndSentToPrintAtBefore(LetterStatus status, Timestamp before);
+    @Query("select l from Letter l where l.status = :status and l.type <> :type and l.sentToPrintAt < :before")
+    Stream<Letter> findStaleLetters(
+        @Param("status") LetterStatus status,
+        @Param("type") String type,
```

Comment:
Do we need to parameterise the type here? I would just hard code the smoke test type here.